templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -1,7 +1,34 @@
 {
   "Comment": "${step_function_name}: A State machine to ingest DR2 BagIt-like packages into Preservica.",
-  "StartAt": "Map metadata",
+  "StartAt": "Validate the metadata.json file",
```

Comment:
Can we call this `Validate input` instead as it should be doing both the `metadataPackage` JSON and the state machine input.